Assets/Scripts/Player/Interaction/PluckItem.cs

```diff
@@ -0,0 +1,36 @@
+using Fusion;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+[RequireComponent(typeof(PickupItem))]
+[RequireComponent(typeof(InteractionController))]
+public class PluckItem : NetworkBehaviour
+{
+
+    private GameObject player;
+    private PickupItem pickupItem;
+
+    private InteractionController interactionController;
+
+    public override void FixedUpdateNetwork()
+    {
+        if (Object.HasStateAuthority && GetInput(out NetworkInputPrototype input))
+        {
+            if (input.ePressed && !pickupItem.itemHeld) {
+                var pickupable = interactionController.closestCollider(player.transform.position, interactionController.pluckables)?.GetComponent<Pluckable>()?.toItem();
+                if (pickupable != null)
+                {
+                    pickupItem.add(pickupable);
+                }
+            }
+        }
+    }

```

Comment:
I think there may be a sensible further refactor here to pull something like "a hand" out from PickupItem, which might see the distinction between a pickupable and a pluckable go down almost to zero -- both would call e.g. `hand.add` instead of pluck telling pickup to add, but I also think for now this setup w/ the interaction controller taking just some of it away is a good place 👍